### ISSUE TITLE
Merge the latest master changes to 5.0-dev

### ISSUE THREAD

**Issue by TomasTurina** [wazuh/wazuh-packages#803]
|Wazuh version|Install type|Action performed|Platform|
|---|---|---|---|
| 5.0-dev | Manager/API/Agent | Install/Upgrade/Remove | All |

There is a problem when executing Jenkins checks when a PR points to `5.0-dev`. The following error appears:

```
    Installed (but unpackaged) file(s) found:
   /var/ossec/tmp/sca-5.0.0-0.commitf941835-tmp/sles/15/cis_sles15_linux.yml
```

This is affecting the following checks:

- Build Linux x86_64 agent rpm package
- Build Linux x86_64 manager rpm package

It seems to be a problem related to missed changes present in `master` branch but not in `5.0-dev` branch.

**Comment by DFolchA** [comment]
## Update report 

**Working branch:** https://github.com/wazuh/wazuh-packages/tree/merge-4.2-master

Merged 4.2 changes into master.

### To Do 
- [ ] Merge master into 5.0-dev

**Comment by DFolchA** [comment]
## Update report 

**Working branch:** https://github.com/wazuh/wazuh-packages/tree/merge-4.2-master

- Test 4.3.0 packages after changes
- Add changes to 5.0.0

### To Do 
- [ ] Test changes in 5.0.0
